blog/Mastering Data Structures in Python.md

```diff
@@ -0,0 +1,331 @@
+---
+title: 'Mastering Data Structures in Python'
+sidebar_label: Data Structures
+authors: [SanthoshSiddhardhaLingamuneni]
```

Comment:
put your info as a author `SanthoshSiddhardhaLingamuneni` in https://github.com/CodeHarborHub/codeharborhub.github.io/blob/main/blog/authors.yml

Reply:
and used github username

Reply:
Made changes